utilities/user_utilities.py

```diff
@@ -1149,6 +1149,31 @@ def get_admin_group_ids():
         __logger.error("Groups not retrievable")
         return []
 
+def overwrite_subevents_to_superevent(overwrite_subevent_list, super_eventid):
+    try:
+        record = find_one(current_app.config['EVENT_COLLECTION'], condition={"_id": ObjectId(super_eventid)})
+        if record:
+            subEvnts = record['subEvents']
```

Comment:
```suggestion
            subEvents = record['subEvents']
```
This could be a typo. Please update the variable name in this method. Thanks.